examples/static-hbase/main.tf

```
@@ -107,10 +108,50 @@ module "emr-hbase" {
   master_ebs_size                 = 50
   core_ebs_size                   = 50
 
-  # Spot Instance definition
-  # on-demand r5.xlarge EC2 + EMR --> $0.252 + $0.063 = $0.315 per Hour
-  core_bid_price = ".100"
-
   # Don't use spot instances for the master nodes
   master_bid_price = ""
+
+  # Security Group IDs
+  emr_managed_master_sg_ids = module.aws-emr-sg-master.security_group_ids
+  emr_managed_core_sg_ids = module.aws-emr-sg-core.security_group_ids
+  emr_service_access_sg_ids = module.aws-emr-sg-service-access.security_group_ids
+}
+
+module "sg-ports" {
+  # source               = "git::https://github.com/Datatamer/terraform-aws-emr.git//modules/aws-emr-ports?ref=6.0.0"
+  source = "../../modules/aws-emr-ports"
+  applications = local.this_application
+}
+
+module "aws-emr-sg-master" {
+  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id = var.vpc_id
+  ingress_cidr_blocks = var.ingress_cidr_blocks
+  egress_cidr_blocks = var.egress_cidr_blocks
+  ingress_ports  = module.sg-ports.ingress_master_ports
+  sg_name_prefix = var.master_name_prefix
```

I don't think we need unique variables for master/core/service. Could do something like this instead since we already have a name related variable for prefixes:
```suggestion
  sg_name_prefix = format("%s-%s", var.name_prefix, "master")
```

---

done